subcommands/devices/list_denied.go

```diff
@@ -0,0 +1,76 @@
+package devices
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/cheynewallace/tabby"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/foundriesio/fioctl/subcommands"
+)
+
+func init() {
+	paginationLimits = []int{10, 20, 30, 40, 50, 100, 200, 500, 1000}
+	limitsStr := ""
+	for i, limit := range paginationLimits {
+		if i > 0 {
+			limitsStr += ","
+		}
+		limitsStr += strconv.Itoa(limit)
+	}
+
+	listCmd := &cobra.Command{
+		Use:   "list-denied",
+		Short: "List device UUIDs that have been denied access to the device-gateway",
+		Run:   doListDenied,
+		Long: `Devices created using a factory-registration-reference server get created
+created on-demand. Because of this, devices are placed into a deny-list when
+they are deleted, so that they can't continue to access the system by getting 
+re-created.`,
+	}
+	cmd.AddCommand(listCmd)
+	listCmd.Flags().IntVarP(&showPage, "page", "p", 1, "Page of devices to display when pagination is needed")
+	listCmd.Flags().IntVarP(&paginationLimit, "limit", "n", 500, "Number of devices to paginate by. Allowed values: "+limitsStr)
+}
+
+func doListDenied(cmd *cobra.Command, args []string) {
+	factory := viper.GetString("factory")
+	logrus.Debugf("Listing denied devices for: %s", factory)
+	assertPagination()
+
+	t := tabby.New()
+	t.AddHeader("UUID", "NAME", "OWNER")
+
+	dl, err := api.DeviceListDenied(factory, showPage, paginationLimit)
+	subcommands.DieNotNil(err)
+	for _, device := range dl.Devices {
+		device.Factory = factory // to make ownerFormatter work
+		t.AddLine(device.Uuid, device.Name, ownerFormatter(&device))
+	}
+	t.Print()
+	if dl.Next != nil {
+		fmt.Print("\nNext page of devices can be viewed with: ")
+		found := false
+		for i := 0; i < len(os.Args); i++ {
+			arg := os.Args[i]
+			if len(arg) > 2 && arg[:2] == "-p" {
+				fmt.Printf("-p%d ", showPage+1)
+				found = true
+			} else if len(arg) == 6 && arg[:6] == "--page" {
```

Comment:
This can come in two forms: `--page <page>` and `--page=<page>`.
Need to handle both.